client/src/components/checkout/purchaseOptions.jsx

```diff
@@ -38,10 +38,10 @@ class purchaseOptions extends React.Component {
 
   addToCart() {
     $.ajax({
-      url: `http://13.52.101.124:80/cart/${this.props.item.item_id}`,
+      url: `/users/0/cart/`,
```

Comment:
Is the user id hardcoded to 1? What if another user wants to add things to their carts?

Reply:
Since the UI doesn't yet have 'Sign in' feature yet, for testing perposes it is assumed user 0 is already signed in. 
Despite that, the checkout API does support many users and is able to function as expected when UI is updated with 'Sign in' feature.